app/pods/print-overviews/decisions/agendaitems/controller.js

```diff
@@ -62,8 +62,11 @@ export default Controller.extend({
       const agendaitem = await this.store.findRecord('agendaitem', agendaitemRow.content.id, {
         include: 'agenda,agenda.created-for',
       });
-      const startDate = agendaitem.get('agenda.createdFor').get('plannedStart');
-      const subcase =  agendaitem.get('agendaActivity.subcase');
+      const agenda = await agendaitem.agenda;
```

Comment:
Nu jullie kleine aanpassingen zelf doen, ontbreekt soms het waarom. Ik vind het oorspronkelijk veel leesbaarder dan de change.

Ik dacht eraan om een utility-functie te schrijven vb. `awaitIncludes`, die uw include-parameter en het gefetchte record meekrijgt en dan een await doet op alle relaties. Ik heb gemerkt dat dit snel leidt tot bugs, vb. bij refresh.

Reply:
Ik ben niet zeker of ik je opmerking correct interpreteer.
- De `get()` zijn verwijderd omdat dat legacy Ember is en sinds Octane niet meer hoeft (met uitzondering van het opvragen van properties van een proxy-object). 

- De `await` zijn toegevoegd om expliciet op de promises te wachten en met de resolved value verder te gaan ipv met de promise. Indien de relatie mee opgenomen is in de `include` kan je er wel vanuit gaan dat die promise onmiddellijk resolved, maar dan kost de `await` ook niets extra om uit te voeren. Voor relaties die niet in de `include` zitten (vb. `agenda-activity`) is de `await` wel noodzakelijk om zeker te zijn dat promise resolved is. 

Het is inderdaad wel zo dat het met die `await` en verschillende variable assignments wat omslachtiger leest, maar het maakt het async gedrag en de mogelijke netwerk requests die de code triggert wel explicieter wat een van de feedback-punten was uit de audit.